### ISSUE TITLE
Existance Check Create User

### ISSUE THREAD

**Issue by pshore73** [sqlcollaborative/docs#48]
Export-DBALogin currently does an "IF USER NOT EXISTS" check when creating logins to the instance, which is great.  However, in generating the TSQL file it does not put a similar check around the create user statement, which when coupled with a restore/refresh we run into errors at times.

My request is simple, can that check be added to the SQL file generation?

Thanks!

**Comment by pshore73** [comment]
Update: The check does exist if you use the -ObjectLevel parameter, so the code to generate is already in the cmdlet. I think it would be useful to check for users in the database regardless of pulling objectlevel permissions or not.

**Comment by lancasteradam** [comment]
@pshore73, I agree with you and I have a proposed fix for this issue. Can you move this issue to https://github.com/sqlcollaborative/dbatools/issues ? This repository is for documentation items.

**Comment by wsmelton** [comment]
Owner or maintainer has to migrate the issue. When I'm back at my computer I'm move it.

**Comment by lancasteradam** [comment]
cool, thanks @wsmelton
